core/pom.xml

```diff
@@ -29,10 +29,28 @@
       <groupId>com.h2database</groupId>
       <artifactId>h2</artifactId>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api -->
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>1.3.2</version>
+    </dependency>
 
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
```

Comment:
Lombok is cool, and can be handy to reduce amount of code. Totaly not against it, and good that you know about that lib. But it's out of scope for that training and there is a big chance that in future you'll have to work with large projects which were built without lombok. So to have some practice, I would ask to not use it for current trail. Modern IDEs have instruments to autogenerate getters, setters and other stuff. And tbh, in this case you won't get much out of it, because creating of POJOs with lombok still require quite a lot of lines of code, especially when they have a lot of properties